.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Found a bug? Tell us and help us improve
+title: ''
+labels: a:bug
+assignees: ''
+
+---
+
+**Does the issue reproduce with the latest available release? (y/n)**
+
+**Your gotify server is running on**
+- [ ] Docker
+- [ ] Linux machine
+- [ ] Windows machine
+<details><summary>Docker startup command or config file here (please mask sensitive information)</summary><br><pre>
```

Comment:
this could be formatted with backticks, I'd think github users are more familiar with backticks than \<pre>
`````
<details>
<summary>Reverse proxy configuration (please mask sensitive information)</summary><br>
  
```
# Add your config here
```
</details>
`````

Reply:
I tried this but I found that markdown is not recognizing backticks correctly inside `details` tags.

Reply:
The empty line before the backticks is required. See f.ex the nextcloud template https://github.com/nextcloud/server/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md